app/upgrades/v1rc7/testnet/upgrade.go

```diff
@@ -0,0 +1,101 @@
+package testnet
+
+import (
+	"context"
+	"fmt"
+
+	sdkmath "cosmossdk.io/math"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/types/query"
+
+	"github.com/babylonlabs-io/babylon/app/keepers"
+	"github.com/babylonlabs-io/babylon/app/upgrades"
+	btcstakingkeeper "github.com/babylonlabs-io/babylon/x/btcstaking/keeper"
+	btcstakingtypes "github.com/babylonlabs-io/babylon/x/btcstaking/types"
+)
+
+const (
+	UpgradeName = "v1rc7"
+)
+
+func CreateUpgrade() upgrades.Upgrade {
```

Comment:
I think you still need to add the call to `v1rc7.CreateUpgrade()` in the file `inlucde_upgrade_testnet.go` to make it available in binary.

Reply:
yes, thanks for reminding me this